fix(light): print an err if trusted store is empty (#4004)
cometbft/cometbft: .changelog/unreleased/bug-fixes/3992-light-panic.md

> - `[light]` Print an error if starting from the trusted store, but it's empty

Reviewer: ```suggestion
- `[light]` Return and log an error when starting from an empty trusted store.
```

>   (due to incorrect configuration or user forgetting to provide a trusted

Reviewer: ```suggestion
  This can happen using the `light` CometBFT command-line command while using a fresh trusted store and no trusted height and hash are provided.
```

>   height and hash arguments).

Reviewer: I forgot to remove this remaining line.
